Add more-info pages for filesystem check unhealthy reasons (#46471)
home-assistant/home-assistant.github.io: source/more-info/unhealthy/data_filesystem_check_error.md

---
title: "Filesystem check failed on the data partition"
description: "More information on what to do when a filesystem check fails on the Home Assistant data partition."
---

## The issue

> At every boot, Home Assistant checks the integrity of its storage partitions. This check failed on the data partition, which means the filesystem is corrupted — files may be missing, unreadable, or in an inconsistent state.

Reviewer: ```suggestion
At every boot, Home Assistant checks the integrity of its storage partitions. This check failed on the data partition, which means the filesystem is corrupted. Files may be missing, unreadable, or in an inconsistent state.
```
We prefer short sentences over the use of em dash. If you do use em dash, use the [Merriam-Webster style](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use), which does not use spaces around the em dash.
https://developers.home-assistant.io/docs/documenting/general-style-guide#language-and-tone

> The data partition stores your configuration, automations, add-ons, and backups. This is most commonly caused by an unexpected power loss or a failing storage device (SD card, USB drive, or eMMC).

Reviewer: ```suggestion
The data partition stores your configuration, automations, apps, and backups. This is most commonly caused by an unexpected power loss or a failing storage device (such as an SD card, USB drive, or eMMC).
```

---
title: "Filesystem check failed on the data partition"
description: "More information on what to do when a filesystem check fails on the Home Assistant data partition."
---

## The issue

At every boot, Home Assistant checks the integrity of its storage partitions. This check failed on the data partition, which means the filesystem is corrupted — files may be missing, unreadable, or in an inconsistent state.

The data partition stores your configuration, automations, add-ons, and backups. This is most commonly caused by an unexpected power loss or a failing storage device (SD card, USB drive, or eMMC).

## The solution

Try rebooting your system first. The filesystem check may be able to repair minor corruption automatically on the next boot attempt.

{% my hardware badge %}

> If the system is still accessible after rebooting, create a full backup immediately and download it to another device before doing anything else.

Reviewer 2: I think we have docs for that, maybe worth linking.

{% my supervisor_backups badge %}

> If the issue persists, you can wipe and reformat the data partition using the CLI:

Reviewer 2: I'd iterate that this deletes all data and the backup must be used to restore afterwards. Also that it is not guaranteed to work if the issue is hardware related (but worth a try...)
